src/prism.c

```diff
@@ -14178,6 +14175,10 @@ parse_arguments(pm_parser_t *parser, pm_arguments_t *arguments, bool accepts_for
                     arguments->block = argument;
                 }
 
+                if (parser->current.type == PM_TOKEN_COMMA) {
```

Comment:
```suggestion
                if (match1(parser, PM_TOKEN_COMMA)) {
```

We use this match1 function in other places, so just updating to match that.

Reply:
Ok, works for me 👍🏼 

Not sure what's up with the rust build.